src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search {
+  display: block;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  width: 100%;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+}
+
+.bigSearch__input {
```

Comment:
You can move this rules to .search selector and make big search as base
![image](https://user-images.githubusercontent.com/54936044/232627182-92a789ab-1f5b-4640-8a8c-56956ce74cf2.png)

Reply:
Thank you for your comments, I hope I all fixed